### ISSUE TITLE
request

### ISSUE THREAD

**Issue by coachshea** [scheibler/khard#19]
I love this product. It has revolutionized the way I work with addresses. There is one feature that I feel would be very useful. As far as I can tell, khard's modify command only works with -s, it would be great if we could alternatively provide the actual location.

**Comment by scheibler** [comment]
Sorry, I don't understand your request.

The "modify" command is for editing your contacts. You may either use it with the "-s" parameter to
search for a specific contact or without it to choose a contact from a list. Afterwards your
favorite text editor opens and you can change your chosen contact's data.

What's the "actual location"? Do you mean the path to the contacts .vcf file and if so, why would
you want to provide that path to khard? At the moment I can't imagine a use case for that.

If you want to open the actual .vcf file, you may use the "source" command:

khard source -s john

Or the same without "-s" to choose from the list.  Maybe that's what you are looking for?

Regards
Eric

**Comment by coachshea** [comment]
Thanks for the quick response. Sorry to be responding to this so late. I had originally responded to this several days ago and just noticed that, for some reason, it didn't upload. 

When I said location, I did mean path, but I did not mean source. Once of the things I love most about this great program is the conversion that takes place with modify and how easy it makes it to alter vcards. My use case was the possibility to combine khard with other tools - for example ack or git-grep - and then pipe the output to khard with xargs. I assume behind the scenes that modify must be doing this anyway. At some point the program must know where to get the source to convert it. If this is an unusual use case, I can live without it, but I thought it would be a nice feature. Thanks again for your consideration and thanks again for the great product.
